containerz/containerz.proto

```diff
@@ -64,6 +80,22 @@ service Containerz {
   // (i.e. kill) a container.
   rpc StopContainer(StopContainerRequest) returns (StopContainerResponse) {}
 
+  // UpdateContainer updates a running container to the image specified in the
```

Comment:
Can you expand on the meaning of this?

Is it fair to say that the combined operation is a rough equivalent to "stop the container, delete the container, launch the container with the new parameters"?

If the container exists but not running at the time the request was received, is it considered an error?

Reply:
Good question - yes this is equivalent and could be implemented with existing RPCs. We introduced this RPC to formalize the contract with the user.

Yes this should be an error - I will update comment. Thanks for pointing that out.

Reply:
Thanks I mentioned that this is the default size.